tests/saw/spec/#output.v#

```diff
@@ -0,0 +1,7731 @@
+
```

Comment:
Was this file meant to be included? What is in it?  Is it possible to add a header comment if it was meant to be included?

Reply:
I don't think this file is necessary. I'll remove it.